### DISCUSSION TITLE
无法正常安装

### DISCUSSION THREAD

**Discussion by laoaideqi** [PenUniverse/PenMods#191] [你问我答]
之前安装过penmods在词典笔上，但是不知道怎么就退出了。这次再安装的时候安装包打开没一会就关了，试了几次都不行

**Comment by notturno-li** [comment]
[penmod1.2.0安装脚本 by Forfuture-h](https://github.com/PenUniverse/PenMods-release/discussions/177)

**Comment by laoaideqi** [comment]
可以使用但是词典笔的ui乱了




?劳埃德·柒?
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "PenUniverse/PenMods-release"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月24日(星期六) 晚上7:25
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [PenUniverse/PenMods-release] 无法正常安装 (Discussion #191)





 
penmod1.2.0安装脚本 by Forfuture-h

**Reply by tofuture-1** [reply]
有图片吗，发一下

**Reply by laoaideqi** [reply]
3520858759加一下qq  这个图片好像发不出去

**Comment by laoaideqi** [comment]
发自我的iPhone


------------------ 原始邮件 ------------------
发件人: forward ***@***.***&gt;
发送时间: 2024年2月24日 22:36
收件人: PenUniverse/PenMods-release ***@***.***&gt;
抄送: laoaideqi ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [PenUniverse/PenMods-release] 无法正常安装 (Discussion #191)





 
有图片吗，发一下

**Comment by laoaideqi** [comment]
![Uploading IMG_3440.jpeg…]()

**Reply by tofuture-1** [reply]
写评论放图片上来的时候要等一下，uploading是上传中
